.gradient_ignore.exs

```diff
@@ -1,19 +1,6 @@
 [
   ## still to be fixed
 
-  # ?
-  {"lib/gradient/elixir_type.ex:70", {:type_error, :mismatch}},
-
-  "lib/gradient/elixir_type.ex:38",
-
-  "lib/gradient/ast_specifier.ex:439",
-
-  "lib/gradient.ex:77",
-
-  "lib/gradient.ex:184",
-
-  "lib/mix/tasks/gradient.ex:106",
```

Comment:
Some of these were changed into magic comments in the files themselves. Others (such as the one on line 5) didn't result in an error when I commented out the ignore, so I just removed them.